src/alfasim_score/converter/alfacase/score_output_generator.py

```diff
@@ -105,11 +112,26 @@ def _generate_output_results(self) -> Dict[str, Any]:
                 )
             },
         }
-        return {
-            "annuli": annuli_output,
-            "MD": measured_depths.tolist(),
-            "production_tubing": production_tubing,
-        }
+        return production_tubing
+
+    def _generate_walls_output(self, results: Results, measured_depths: np.ndarray) -> Dict[str, Any]:
+        """Create data for the output results of walls."""
+        walls_output: Dict[str, Any] = {}
+        wall_index = 0
+        for wall_label in self.walls:
```

Comment:
You can use the reverse iterator:
```
for wall_label in reversed(self.walls):
```
 or slice
```
for wall_label in self.walls[::-1]:
```

Reply:
Nice tip, thx!